wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -255,7 +255,7 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
                 crossAxisAlignment: CrossAxisAlignment.stretch,
                 children: [
                   SizedBox(
-                    height: 60.h,
+                    height: 75.h,
```

Comment:
Whats the reason for this change?

Reply:
@kjawadDeveloper2  It is showing a little overflow problem in 2 mobile screens, that is why  I increased height little to work smoothly in every screen.